roles/setup/tasks/setup-container.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Enable linger for {{ receptor_user }}
+  become: true
+  ansible.builtin.command:
+    cmd: "loginctl enable-linger {{ receptor_user }}"
+  changed_when: false
+
+- name: Get UID of {{ receptor_user }}
+  ansible.builtin.command:
+    cmd: "id -u {{ receptor_user }}"
+  register: receptor_uid
+  changed_when: false
+
+- name: Setup container
+  become: true
+  become_user: "{{ receptor_user }}"
+  environment:
+    XDG_RUNTIME_DIR: "/run/user/{{ receptor_uid.stdout }}"
+  vars:
+    receptor_uid_directory: "/run/user/{{ receptor_uid.stdout }}"
+  block:
+    - name: Enable podman socket
+      ansible.builtin.systemd:
+        name: podman.socket
+        state: started
+        enabled: true
+        scope: user
+
+    - name: Pull the latest release of receptor from Quay
+      containers.podman.podman_image:
+        name: "{{ receptor_image_name }}"
+        tag: "{{ receptor_image_tag }}"
+        state: present
+
+    - name: Run receptor image
+      containers.podman.podman_container:
+        name: "{{ receptor_service_name }}"
+        image: "{{ receptor_image_name }}:{{ receptor_image_tag }}"
+        state: created
+        network: host
+        pid: private
+        pids_limit: 0
+        cap_drop:
+          - CAP_AUDIT_WRITE
+          - CAP_MKNOD
+        user: root
+        volume:
+          - "{{ receptor_config_dir }}/receptor.conf:{{ receptor_config_dir }}/receptor.conf:ro"
+          - "{{ receptor_worksign_public_keyfile }}:{{ receptor_worksign_public_keyfile }}:ro"
+          - "{{ receptor_tls_certfile }}:{{ receptor_tls_certfile }}:ro"
+          - "{{ receptor_tls_keyfile }}:{{ receptor_tls_keyfile }}:ro"
+          - "{{ receptor_ca_certfile }}:{{ receptor_ca_certfile }}:ro"
+          - "{{ receptor_socket_dir }}:{{ receptor_socket_dir }}:z"
+          - "{{ receptor_uid_directory }}/podman/podman.sock:/run/podman/podman.sock:z"
+        privileged: true
+        security_opt:
+          - 'label=disable'
+        env:
+          RECEPTORCTL_SOCKET: '{{ receptor_socket_dir }}/{{ receptor_control_filename }}'
+        command: 'receptor --config {{ receptor_config_dir }}/receptor.conf'
+        generate_systemd:
+          container_prefix: ''
+          path: ~/.config/systemd/user/
+          separator: ''
+
+    - name: Ensure receptor service is started
+      ansible.builtin.systemd_service:
+        name: '{{ receptor_service_name }}'
+        state: started
+        enabled: true
+        scope: user
+        daemon_reload: true
+
+- name: Add 'receptorctl' alias
+  become: true
+  become_user: "{{ ansible_user }}"
```

Comment:
`ansible_user` or `receptor_user` ?

Reply:
I was under impression we want to add the alias as the `ansible_user`, which would be `ec2-user` in the AWS case. I can instead add it to the `receptor_user` which would be `awx`

The `install_receptor` playbook will create the `awx` user. This alias for the `ec2-user`, is able to login to the awx user, exec into the container, and run `receptor` commands from there.

Reply:
Resolving since no longer adding alias